web/themes/custom/lex/old_scss/styles.sass

```diff
@@ -14,6 +14,7 @@
 @import '_search_page'
 @import '_lexcalendar'
 @import '_maintenance_page'
+@import '_formstack-overrides'
```

Comment:
This was added in to account for the forms in the sidebar. Not sure why it was never there.